stubs/inertia/resources/js/Pages/Profile/Show.vue

```diff
@@ -20,11 +20,17 @@
                     <two-factor-authentication-form class="mt-10 sm:mt-0" />
                 </div>
 
+                <div v-if="$page.props.jetstream.hasSocialiteFeatures">
+                    <jet-section-border />
+
+                    <remove-connected-accounts-form :accounts="connectedAccounts" class="mt-10 sm:mt-0" />
+                </div>
+
                 <jet-section-border />
 
                 <logout-other-browser-sessions-form :sessions="sessions" class="mt-10 sm:mt-0" />
 
-                <template v-if="$page.props.jetsteam.hasAccountDeletionFeatures">
+                <template v-if="$page.props.jetstream.hasAccountDeletionFeatures">
```

Comment:
I think we already got a PR for this. Might need to merge 1.x into master for this. If not, can you send this in separately to 1.x?